tests/virt/upgrade_custom/vgpu/test_vgpu_vm_upgrade.py

```
@@ -0,0 +1,49 @@
+import pytest
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)